### ISSUE TITLE
Could it support 4dof optimization?

### ISSUE THREAD

**Issue by forlinchow** [isl-org/Open3D#5762]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### My Question

pose_graph_optimization, Can it support 4dof optimization?

**Comment by v-parmar** [comment]
Yes. 4-DOF pose graph optimization is performed to correct drifts and achieve global consistent. The 4-DOF contains x, y, z, and yaw angle, which is the actual drifted direction in the visual-inertial system.

**Comment by forlinchow** [comment]
> Yes. 4-DOF pose graph optimization is performed to correct drifts and achieve global consistent. The 4-DOF contains x, y, z, and yaw angle, which is the actual drifted direction in the visual-inertial system.

Is there a link to the document? Don't see how to use 4dof here

**Comment by forlinchow** [comment]
@v-parmar
